homeassistant/components/weather/openweathermap.py

```diff
@@ -129,10 +131,22 @@ def attribution(self):
         """Return the attribution."""
         return ATTRIBUTION
 
+    @property
+    def forecast(self):
+        """Return the forecast array."""
+        temp_unit = ('celsius' if self.hass.config.units.is_metric else
+                     'fahrenheit')
+
+        return [{'datetime': entry.get_reference_time('iso'),
+                 'temp': entry.get_temperature(temp_unit).get('temp', None)}
```

Comment:
I just thought about the fact that the keys here might need to come from the Weather Component itself (as constants), to make it easier to extend, any thought on this?